CMakeLists.txt

```diff
@@ -36,128 +36,98 @@ option(BUILD_GDASBUNDLE "Build GDAS Bundle" ON)
-  ecbuild_bundle( PROJECT mom6 GIT "https://github.com/jcsda-internal/MOM6.git" BRANCH main-ecbuild RECURSIVE )
-  ecbuild_bundle( PROJECT soca GIT "https://github.com/jcsda-internal/soca.git" BRANCH develop )
-
-  # Build JEDI/DA or other peripherals
-  ecbuild_bundle( PROJECT gdas-utils SOURCE "./utils" )
-
-# Build IODA converters
-  option(BUILD_IODA_CONVERTERS "Build IODA Converters" ON)
-  if(BUILD_IODA_CONVERTERS)
-    ecbuild_bundle( PROJECT iodaconv GIT "https://github.com/JCSDA-internal/ioda-converters.git" BRANCH develop )
-  endif()
-
-# Land associated repositories
-  ecbuild_bundle( PROJECT land-imsproc  GIT "https://github.com/NOAA-PSL/land-IMS_proc.git"  TAG 6373819 )
-  ecbuild_bundle( PROJECT land-jediincr GIT "https://github.com/NOAA-PSL/land-apply_jedi_incr.git"  TAG 2923344)
-
-# ioda, ufo, fv3-jedi, and saber test data
-#---------------------------------
-if(CLONE_JCSDADATA)
-
-  # If IODA branch is being built set GIT_BRANCH_FUNC to IODA's current branch.
-  # If a tagged version of IODA is being built set GIT_TAG_FUNC to ioda's current tag. In this case,
-  # IODA test files will be download from UCAR DASH and ioda-data repo will not be cloned.
-  # When LOCAL_PATH_JEDI_TESTFILES is set to the directory of IODA test files stored
-  # in a local directory, ioda-data repo will not be cloned
-
-    find_branch_name(REPO_DIR_NAME ioda)
-  # When LOCAL_PATH_JEDI_TESTFILES is set to the directory of IODA test files stored
-  # in a local directory, ioda-data repo will not be cloned
-    if( NOT DEFINED ENV{LOCAL_PATH_JEDI_TESTFILES} AND NOT DEFINED GIT_TAG_FUNC )
-      ecbuild_bundle( PROJECT ioda-data GIT "https://github.com/JCSDA-internal/ioda-data.git" BRANCH develop )
-
-      # If IODA's current branch is available in ioda-data repo, that branch will be checked out
-      branch_checkout (REPO_DIR_NAME ioda-data
-                      BRANCH ${GIT_BRANCH_FUNC} )
-    endif()
-
-  # same procedure for ufo-data
-    find_branch_name(REPO_DIR_NAME ufo)
-    if( NOT DEFINED ENV{LOCAL_PATH_JEDI_TESTFILES} AND NOT DEFINED GIT_TAG_FUNC )
-      ecbuild_bundle( PROJECT ufo-data GIT "https://github.com/JCSDA-internal/ufo-data.git" BRANCH develop )
-
-      # If UFO's current branch is available in ioda-data repo, that branch will be checked out
-      branch_checkout (REPO_DIR_NAME ufo-data
-                      BRANCH ${GIT_BRANCH_FUNC} )
-    endif()
-
-  # same procedure for fv3-jedi-data
-    find_branch_name(REPO_DIR_NAME fv3-jedi)
-    if( NOT DEFINED ENV{LOCAL_PATH_JEDI_TESTFILES} AND NOT DEFINED GIT_TAG_FUNC )
-      ecbuild_bundle( PROJECT fv3-jedi-data GIT "https://github.com/JCSDA-internal/fv3-jedi-data.git" BRANCH develop )
-
-      # If fv3-jedi's current branch is available in ioda-data repo, that branch will be checked out
-      branch_checkout (REPO_DIR_NAME fv3-jedi-data
-                      BRANCH ${GIT_BRANCH_FUNC} )
-    endif()
-
-  endif(CLONE_JCSDADATA)
-
-endif(BUILD_GDASBUNDLE)
+## Initialize bundle
```

Comment:
Lines 39 to 130 are not executed.  Do we need to retain these commented out lines?

Reply:
Huh, I thought I pushed a commit removing these lines... guess I didn't

Reply:
I had to push an almost empty commit but now this should be resolved. Weird.